client/src/components/boxes/DepositBox/DepositBoxForm/DepositBoxForm.tsx

```diff
@@ -139,7 +133,7 @@ export const DepositBoxForm = () => {
               })}
             <Space className={s.sum}>
               <>Suma</>
-              <>{acc.toFixed(2).toString() + ' zł'}</>
+              <>{total.toFixed(2).toString() + ' zł'}</>
```

Comment:
Tutaj wyświetlana suma powinna być tylko z PLN, bez walut obcych. Robię review na telefonie więc nie wiem czy tak jest, ale upewnij się proszę i dodaj odpowiedni komentarz.

Przyczyną jest taka, że waluty obce są rozliczane w inny sposób, dlatego nie są tu pokazywane. Chodzi też o to żeby wolontariusz się nie pomylił przy przepisywaniu sumy na kartce.

Reply:
Jest to suma jedynie PLN obce tu nie są sumowane, dodam odpowiedni komentarz by wszystko było jasne